Adafruit_DAP.h

```diff
@@ -205,91 +211,87 @@ class Adafruit_DAP {
 // DAP for SAM
 class Adafruit_DAP_SAM : public Adafruit_DAP {
 public:
-  Adafruit_DAP_SAM(void){};
+  Adafruit_DAP_SAM(void) : locked(false) {};
   ~Adafruit_DAP_SAM(void){};
 
+  static const size_t PAGESIZE = SAM_PAGE_SIZE;
+  static const size_t USER_ROW_SIZE = 256;
   static device_t devices[];
-  device_t target_device;
```

Comment:
Moved to base class, since all child classes should set it in select(...) method